### PR TITLE
JBR-5221 Add announcing to JBRAPI

### PR DESCRIPTION
**Pull request by savoptik**

### CODE REVIEW

File: src/java.desktop/share/classes/com/jetbrains/desktop/JBRApiModule.java [hunk 1]
```diff
@@ -40,6 +40,8 @@ public class JBRApiModule {
                 .clientProxy("java.awt.Desktop$DesktopActionsHandler", "com.jetbrains.DesktopActions$Handler")
                 .service("com.jetbrains.ProjectorUtils")
                     .withStatic("overrideGraphicsEnvironment", "overrideLocalGraphicsEnvironment", "java.awt.GraphicsEnvironment")
-                    .withStatic("setLocalGraphicsEnvironmentProvider", "setLocalGraphicsEnvironmentProvider", "java.awt.GraphicsEnvironment");
+                    .withStatic("setLocalGraphicsEnvironmentProvider", "setLocalGraphicsEnvironmentProvider", "java.awt.GraphicsEnvironment")
+                .service("com.jetbrains.AccessibleAnnouncer")
+                .withStatic("announce", "announce", "sun.swing.AccessibleAnnouncer");
```
Comment (YaaZ) [inline src/java.desktop/share/classes/com/jetbrains/desktop/JBRApiModule.java:45]: `withStatic` in JBR API is related to the previous `service` call and is padded with 4 extra spaces, please add padding for consistency.

File: src/jetbrains.api/src/com/jetbrains/AccessibleAnnouncer.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    /*CONST sun.swing.AccessibleAnnouncer.ANNOUNCE_**/
+
+        /**
+         * This method makes an announcement with the specified priority from an accessible to which the announcing relates
+         *
+         * @param a      an accessible to which the announcing relates
+         * @param str      string for announcing
+         * @param priority priority for announcing
+         */
+        void announce(Accessible a, final String str, final int priority);
```
Comment (YaaZ) [inline src/jetbrains.api/src/com/jetbrains/AccessibleAnnouncer.java:45]: `announce` with its javadoc has too much padding, please reformat the file.

**Review by YaaZ — COMMENTED** [review]
Looks good to me except formatting.

File: src/java.desktop/share/classes/com/jetbrains/desktop/JBRApiModule.java (hunk 1 above)
Comment (savoptik) [reply src/java.desktop/share/classes/com/jetbrains/desktop/JBRApiModule.java]: Done.

File: src/jetbrains.api/src/com/jetbrains/AccessibleAnnouncer.java (hunk 2 above)
Comment (savoptik) [reply src/jetbrains.api/src/com/jetbrains/AccessibleAnnouncer.java]: Done.

**Review by YaaZ — APPROVED** [review]

**Pull request closed without merging**